### DISCUSSION TITLE
Full Data for player IDs Competition IDs and club IDs

### DISCUSSION THREAD

**Discussion by charjn03** [dcaribou/transfermarkt-datasets#157] [Q&A] [answered]
I would like to understand if your data contains the full data on the site. I've only been able to locate a sample of the site data.

**Comment by dcaribou** [comment, marked as answer]
Hey @charjn03 
Although the scraping code has the potential to scrap any entity (player, competition, club, appearance, game) in the website, for practical reasons the curated dataset only contains a subset of them The subset is defined by the list of competitions provided in [this](https://github.com/dcaribou/transfermarkt-datasets/blob/master/data/competitions.json) file.

If you are interested on the data for a competition that is not there, it may be possible to add it to the list though.
